test/cpp/qps/report.h

```diff
@@ -103,6 +104,27 @@ class GprLogReporter : public Reporter {
   void ReportTimes(const ScenarioResult& result) const GRPC_OVERRIDE;
 };
 
+/** Reporter for client leaderboard. */
+class PerfDbReporter : public Reporter {
+ public:
+  PerfDbReporter(const string& name, const string& access_token, const string& test_name, const string& sys_info, const string& server_address)
+   : Reporter(name), access_token_(access_token), test_name_(test_name), sys_info_(sys_info) {
+    perfDbClient.init(grpc::CreateChannel(server_address, grpc::InsecureCredentials(), ChannelArguments()));
+  }
```

Comment:
Ok, now I understand the need for the mutable field, const functions, etc. For future reference, the PerfDBClient functions and fields should have been normal and the perfDbClient_ in this object labeled mutable. But that is only if we ever accept mutable into grpc, and I think that we shouldn't. @dgquintas has recently put in a PR to remove the const specification from many of these functions, and this should get merged soon.